Task10_Auth/src/main/java/reseponsesDTO/UsersDto.java

```diff
@@ -43,4 +47,42 @@ public String getZipCode() {
     public void setZipCode(String zipCode) {
         this.zipCode = zipCode;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof UsersDto)) return false;
+        UsersDto usersDto = (UsersDto) o;
+        return getName().equals(usersDto.getName()) &&
+                getUserSex() == usersDto.getUserSex();
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(getName(), getUserSex());
+    }
+
+    public UsersDto createUserDtoObject(int age, String name, UsersDto.sex sex, String zipCode) {
+        UsersDto usersDto = new UsersDto();
+        usersDto.setAge(age);
+        usersDto.setName(name);
+        usersDto.setUserSex(sex);
+        usersDto.setZipCode(zipCode);
+        return usersDto;
+    }
+
+    public UsersDto createUserDtoObject(String name, UsersDto.sex sex) {
+        UsersDto usersDto = new UsersDto();
+        usersDto.setName(name);
+        usersDto.setUserSex(sex);
+        return usersDto;
+    }
```

Comment:
Looks strange, make these methods as constructors or make them static

Reply:
How can I fill not all fields, If I will create a constructor with some default values.

For example, when I need only name and sex?

Reply:
You can have different ctors with different parameters (frankly speaking it is the main idea of ctors). Current solution: you're creating object to create another object of the same class, it is odd

Reply:
done